### ISSUE TITLE
Git clone not working for sample projects or by using a custom git repo url

### ISSUE THREAD

**Issue by nabiltntn** [eclipse-che/che#13151]
### Description

All git functionalities from IDE are not working like creating a workspace with a sample project or by providing a git repo.
It's not working too from within a Workspace by doing 'import project'

### Reproduction Steps
- Create workspace
- Select a sample node project with a Node stack for example.
- Workspace starts but git clone operation does not finish with alert message `Importing project ...`

**OS and version:**   
- 6.9.0 and 7.0.0.beta
- OpenShift with multi-user
- Self-signed-certificate
- Custom HTTP_PROXY and HTTPS_PROXY env variables to access internet

**Diagnostics:** 
I am able to clone a project using the `git clone` command from a Workspace terminal.

**Comment by nabiltntn** [comment]
On the browser console, i have this error message :

```
ERROR (org.eclipse.che.plugin.pullrequest.client.ContributionMixinProvider): Failed to get VCS hosting service nodejs-hello-world. Cause: org.eclipse.che.api.promises.client.FunctionException: org.eclipse.che.plugin.pullrequest.client.vcs.hosting.NoVcsHostingServiceImplementationException: No implementation of the VcsHostingService for the current project
```

**Comment by nabiltntn** [comment]
Looks the same as https://github.com/eclipse/che/issues/12267
I guess the fix is not compatible with Openshift and K8s

**Comment by eivantsov** [comment]
@nabiltntn the trace you see in browser console does not seem to cause the problem you see.

I suspect there might be some proxy issuer perhaps? Can you open a terminal and git clone manually?

**Comment by nabiltntn** [comment]
Yes, i was able to do it as i mentioned in the issue both for Github as for internal Gitlab Repos

**Comment by nabiltntn** [comment]
This a snapshot , i guess the connection was established correctly because the project tree was able to tell that it's the `master` branch

![Capture d’écran 2019-04-24 à 15 38 46](https://user-images.githubusercontent.com/223719/56663817-3a962200-66a7-11e9-9790-63ae0c984a86.png)

**Comment by diogoburgosr** [comment]
I'm running through similar problem using the container 'eclipse/che-server:7.0.0-beta-5.0'. 
I'm behind a corporate proxy with username and password (<username>:<password>@<host>:<port>). 

I have started a 'robertdebock/docker-cntlm' container to bypass the username:password config and set Che proxy to cntlm <host>:<port>. 
Cntlm container works fine but web IDE cannot clone a sample project for Java Workspace. Tried importing it as well without success. Cntlm logs don't show requests to GitHub from it.

Talking to people at Mattermost Eclipse-che channel, they told me the IDE uses JGit version 4.9.0.201710071750-r. 
I have cloned this JGit version, built it, and used the JGit Cli inside che-server container to clone a project. It worked fine. Cntlm got the request: _cntlm[22]: 172.17.0.1 CONNECT github.com:443_

Is that really a bug or am I missing something?
